### PR TITLE
Added scripts for jenkins builds

### PR DESCRIPTION
**Pull request by udaysagar2177**

### CODE REVIEW

File: sfx_scripts/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:14.10
+
+RUN apt-get update
+RUN apt-get -y install dpatch build-essential devscripts fakeroot quilt dh-make automake libdistro-info-perl autoconf flex bison libtool pkg-config pbuilder debootstrap wget git-core debhelper po-debconf autotools-dev libltdl-dev libgcrypt11-dev libglib2.0-dev
+
+CMD "/bin/bash" "/opt/mount/input/collectd-build/sfx_scripts/cmdseq.sh"
```
Comment (tedoc2000) [inline sfx_scripts/Dockerfile]: Make cmdseq.sh executable and then there is no need to /bin/bash it

File: sfx_scripts/cmdseq.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+mkdir /tmp/input/ &&
```
Comment (tedoc2000) [inline sfx_scripts/cmdseq.sh]: Use a name other than "mount" .

It seems like you need
/opt/collectd-source
/opt/collectd-build-source

Also there is no need to use /tmp. How about /opt/collectd-build

File: sfx_scripts/Dockerfile (hunk 1 above)
Comment (udaysagar2177) [inline sfx_scripts/Dockerfile]: fixed it

File: sfx_scripts/cmdseq.sh (hunk 2 above)
Comment (udaysagar2177) [inline sfx_scripts/cmdseq.sh]: I can explain about this on Monday

**Pull request merged**